service/controller/app/key/key.go

```diff
@@ -137,6 +138,18 @@ func KubecConfigSecretNamespace(customResource v1alpha1.App) string {
 	return customResource.Spec.KubeConfig.Secret.Namespace
 }
 
+func LabelSelectorService() labels.Selector {
+	// Selector to check if "app=chart-operator" and if helm-major-version is not belonging to 3.
+	s := fmt.Sprintf("%s=%s,%s notin (%d)", label.App, "chart-operator", label.HelmMajorVersion, 3)
```

Comment:
It will only get chart-operator app CR with no match for helm 3 version labels.

Reply:
This would be easier to review if all the deletions were in one PR and have a separate for the label selector.

Reply:
Didn't notice that was including the part of all together, sorry!